src/c++/perf_analyzer/test_profile_data_exporter.cc

```diff
@@ -75,7 +78,8 @@ TEST_CASE("profile_data_exporter: ConvertToJson")
               {
                 "timestamp" : 1,
                 "sequence_id" : 1,
-                "response_timestamps" : [ 2, 3 ]
+                "response_timestamps" : [ 2, 3 ],
+                "response_outputs" : [ "response_output1", "response_output2" ]
```

Comment:
Moving the conversation into a thread:

Me:
> The example report you posted looks great! However, it looks like it should also include some information to associate the responses with their requests. Are there plans to include either the corresponding inputs or request_ids etc. to correlate them later? Do we think that would be useful?

Elias:

> @rmccorm4 the way the json is structured now, we have the responses contained within their respective request. Originally we thought this would be sufficient to enable correlation of 1 request to N responses. Do you think we should have a different approach going forward?

Reply:
@debermudez I see the objects are within a nested list of `requests` - but how do we actually know what requests those were? What was the request_id or input of each request? 

ex: How do we know "machine learning is a very important part of the business." is the response to the question "what is machine learning?"

Reply:
OH i understand.
I dont think requests are in scope for this work right now.
We have talked about it but it will be a future release.